9.2/paper.tex

```diff
@@ -246,8 +246,22 @@ \section{Overview}
   example for this is to solve the time-harmonic version of the
   Maxwell equations that has the electric and magnetic fields as
   solution. This, too, has been addressed in this release.
-  \item \todo[inline]{Reza: Please add a short summary of the
-    \texttt{DiscreteTime} class}
+\item The class \texttt{DiscreteTime} is introduced to provide a more
```

Comment:
You need to learn to not use the passive voice as much :-) The new class wasn't just introduced -- *you* (or, for the purposes of this paper, the deal.II authors) added the class.